Bio/Align/Applications/_Guidance2.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 by Rob Gilmore and Shaurita Hutchins. All rights reserved.
+# Based on ClustalOmega wrapper copyright 2011 by Andreas Wilm.
+#
+# Wrapper for Guidance2 by Rob Gilmore (2017).  http://guidance.tau.ac.il/ver2/
+# Used _ClustalOmega.py as template.
+#
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+"""Command line wrapper for the multiple alignment program, GUIDANCE2.
+It weights, filters or masks unreliably aligned positions in multiple sequence alignments.
+"""
+
+from __future__ import print_function
+
+import os
+from Bio.Application import _Option, AbstractCommandline
+
+
+class Guidance2Commandline(AbstractCommandline):
+    u"""Command line wrapper for GUIDANCE2.
+
+    http://guidance.tau.ac.il/ver2/
+
+    Example:
+    --------
+
+    >>> from Bio.Align.Applications import Guidance2Commandline
+    >>> import os
+    >>> seqFile = "HTR1A.ffn"
+    >>> msaProgram = "CLUSTALW"
+    >>> seqType = "codon"
+    >>> outDir = Path(os.getcwd)
+    >>> Guidance2_cline = Guidance2Commandline(seqFile, msaProgram, seqType, str(outDir), bootstraps=20, seqCutoff=0.63,\
+                                    colCutoff=0.9, outOrder='as_input', dataset='HTR1A')
+    >>>> print(Guidance2_cline)
+    perl guidance.pl --seqFile HTR1A.ffn --msaProgram CLUSTALW --seqType codon --outDir C://Users/rgilmore/HTR1A
```

Comment:
This output dir value is going to be a problem for a doctest. Can you stick to a relative path?

Note to include the doctest in our test suite, it needs adding to the list in ``Tests/run_tests.py``